lib/auth.js

```diff
@@ -139,7 +126,7 @@ module.exports = new Model({
         }.bind(this));
       } else {
         console.log('Registering new account', given.login);
-        var registrationRequest = this._getAuthToken().then(function(token) {
+        var registrationRequest = getCSRFToken(url, JSON_HEADERS, apiClient).then(function(token) {
```

Comment:
The `getCSRFToken` function doesn't take params... was this an intended refactor or in error?

Reply:
We can just require in the `apiClient` and the config file after the `JSON_HEADERS` move to it. The `url` is undefined here, but the `getCSRFToken` function defines it within.